### PR TITLE
Fix issue where notes directly in a notebook could not be unlinked

### PR DESCRIPTION
**Pull request by alihamuh**
note can be unlinked.

### CODE REVIEW

File: packages/core/collections/notes.js [hunk 1]
```diff
@@ -347,11 +347,11 @@ export default class Notes extends Collection {
 
     for (const noteId of noteIds) {
       const note = this.note(noteId);
-      if (!note || note.deleted || !note.notebooks) {
-        continue;
-      }
 
       if (topicId) {
+        if (!note || note.deleted || !note.notebooks) {
```
Comment (thecodrr) [inline packages/core/collections/notes.js:352]: We should only check for `!note.notebooks` here. All other checks should remain where they were.

**Review by thecodrr — CHANGES_REQUESTED** [review]

File: packages/core/collections/notes.js (hunk 1 above)
Comment (alihamuh) [reply packages/core/collections/notes.js:352]: okay

File: packages/core/collections/notes.js [hunk 2]
```diff
@@ -347,11 +347,15 @@ export default class Notes extends Collection {
 
     for (const noteId of noteIds) {
       const note = this.note(noteId);
-      if (!note || note.deleted || !note.notebooks) {
+
+      if (!note || note.deleted) {
```
Comment (thecodrr) [inline packages/core/collections/notes.js:351]:
```suggestion
      if (!note || note.deleted) {
```

File: packages/core/collections/notes.js [hunk 3]
```diff
@@ -347,11 +347,15 @@ export default class Notes extends Collection {
...
         continue;
       }
 
       if (topicId) {
+        if (!note.notebooks) {
+          continue;
+        }
```
Comment (thecodrr) [inline packages/core/collections/notes.js:358]:
```suggestion
        if (!note.notebooks) continue;
```

**Review by thecodrr — CHANGES_REQUESTED** [review]

**Pull request merged**